src/js/components/Layer/__tests__/Layer-test.js

```diff
@@ -212,4 +212,34 @@ describe('Layer', () => {
     ref.current.componentDidMount();
     expect(queryByTestId(document, 'test-layer-container')).toMatchSnapshot();
   });
+
+  test('focus on layer', () => {
+    render(
+      <Grommet>
+        <Layer data-testid="focus-layer-test">
+          <input />
+        </Layer>
+      </Grommet>,
+    );
+
+    const layerNode = getByTestId(document, 'focus-layer-test');
+    expect(layerNode).toMatchSnapshot();
+    expect(document.activeElement.nodeName).toBe('A');
```

Comment:
That's probably OK.

It might be slightly more robust to add an `<input autofocus />` after the `Layer`. Just a thought.

Reply:
Yes, that would make it more hostile environment, so I added it.

Reply:
Notify @ericsoderberghp